influx.go

```diff
@@ -408,7 +433,7 @@ func addIDB(ocData *na_pb.OpenConfigData, jctx *JCtx, rtime time.Time) {
 			jLog(jctx, fmt.Sprintf("Sending %d points to batch channel for path: %s\n", len(points), ocData.Path))
 			for i := 0; i < len(points); i++ {
 				jLog(jctx, fmt.Sprintf("Tags: %+v\n", points[i].Tags()))
-				if f, err := points[i].Fields(); err != nil {
+				if f, err := points[i].Fields(); err == nil {
```

Comment:
Are we planning to print the log even if err is Nil

Reply:
Yes @balasankarrajaguru, we are printing KV's if verbose logging is on. It prints received KVs.